src/models/multiBarChart.js

```diff
@@ -429,8 +674,25 @@ nv.models.multiBarChart = function() {
             rightAlignYAxis = _;
             yAxis.orient( rightAlignYAxis ? 'right' : 'left');
         }},
+        xTickFormat: {get: function(){return xAxis.xTickFormat();}, set: function(_){
```

Comment:
I still have the opinion that this feature does not make much sense with this chart. So don't worry too much about fixing these